pkgs/default.nix

```diff
@@ -288,4 +289,9 @@ in rec {
 
     NIX_CFLAGS_COMPILE = "-I${imagemagick}/include/ImageMagick-6";
   };
+
+  virtualbox = linuxPackages: (newScope linuxPackages) ./virtualbox {
```

Comment:
If there are ever multiple linuxPackages though it may make sense to put them all in one set.